roles/custom/matrix-client-element/templates/config.json.j2

```diff
@@ -36,6 +36,9 @@
 	"jitsi": {
         "preferredDomain": {{ matrix_client_element_jitsi_preferredDomain|to_json }}
     },
+	{% endif %}
+	{% if matrix_client_element_location_sharing_enabled %}
+	"map_style_url": "https://{{ matrix_server_fqn_element }}/style.json",
```

Comment:
Can we rename this file to `map_style.json`? Seems like it's more self-explanatory and less likely to conflict with something else in the future.

Reply:
Sure, I have renamed `style.json` to `map_style.json`. It works fine.
Also I found out that you can define `map_style_url` in /.well-known/matrix/client which takes precedence if is found there. And added the same url to `map_style.json` also there.